source/utilities.cc

```diff
@@ -60,6 +60,148 @@ namespace aspect
    */
   namespace Utilities
   {
+    namespace internal
+    {
+      namespace MPI
+      {
+#ifdef DEAL_II_WITH_MPI
```

Comment:
isn't this already available in deal.II in mpi.templates.h?

Reply:
Not sure as it is an internal detail right now. Maybe we should promote this from internal to external in deal.II. Maybe leave a comment about this fact here?